os/board/rtl8730e/src/rtl8730e_logtask.c

```diff
@@ -0,0 +1,98 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <debug.h>
+#include <assert.h>
+#include <errno.h>
+
+#include <tinyara/irq.h>
+#include <osdep_service.h>
+#include "rtk_km4log.h"
+#include "osif.h"
+
+/* -------------------------------- Defines --------------------------------- */
+#ifndef CONFIG_RTL8730E_KM4_LOGTASK_PRIO
+#define CONFIG_RTL8730E_KM4_LOGTASK_PRIO 100
+#endif
+
+#ifndef CONFIG_RTL8730E_KM4_LOGTASK_STACK
+#define CONFIG_RTL8730E_KM4_LOGTASK_STACK 512
+#endif
+/* ------------------------------- Data Types ------------------------------- */
+
+/* -------------------------- Function declaration -------------------------- */
+
+/* ---------------------------- Global Variables ---------------------------- */
+void * g_km4_log_queue = NULL;
+
+/* --------------------------- Private Variables ---------------------------- */
+static struct task_struct km4_log_task;
+
+void rtl8730e_km4_logtask(void)
+{
+	static km4log_msg_t event = { 0 };
+
+	/* initialize the queue that will hold the messages */
+	if (!osif_msg_queue_create(&g_km4_log_queue, CONFIG_KM4_MAX_LOG_QUEUE_SIZE, sizeof(km4log_msg_t))) {
+		DBG_8195A("Fail to init km4/np log msg queue\n");
+		g_km4_log_queue = NULL;	// ensure that checking against this handle will be NULL
+		return;
+	}
+
+	while(true) {
+		/* loop and consume an item from the queue to print */
+		if (osif_msg_recv(g_km4_log_queue, &event, 0xFFFFFFFF)) {
+			/* prevent accidental buffer overflow when printf */
+			((uint8_t *)event.buffer)[CONFIG_KM4_MAX_LOG_BUFFER_SIZE - 1] = 0;
+			dbg_noarg("%s", event.buffer);
+#ifndef CONFIG_AMEBASMART_USBDEVICE
+			/* only for UART-only operation due to lower baudrate
+			 * ensure no mixlog due to other task buffering prints by giving some cpu time. 
+			 * setting this too low will bootloop, 1us is unsafe
+			 */
+			rtw_usleep_os(10);										
+#endif
+			/* set the first byte to null to cause string to print empty in case this buffer slot is accidentally reused */
+			((uint8_t *)event.buffer)[0] = 0;
+		}
+	}
+}
+
+void rtl8730e_km4_logtask_initialize(void)
+{
+	static bool initialized = 0;
+
+	if (!initialized) {
+		/* create the log consumption task */
+		if (false == rtw_create_task(&km4_log_task, (const char *)"km4_log_task", CONFIG_RTL8730E_KM4_LOGTASK_STACK, CONFIG_RTL8730E_KM4_LOGTASK_PRIO, (void *)rtl8730e_km4_logtask, NULL)) {
+			DBG_8195A("Fail to create init km4/np logtask\n");
+			return;
+		}
+
+		lldbg("Logtask init ok!\n");
```

Comment:
The os_start function calls the `up_initialize` function first which includes the `up_serialinit` and then calls the `os_bringup`. The `os_bringup` calls the `up_boardinitialize` and this task lauching is there.
SO, you should not use the `lldbg` here. 
The `lldbg` must be used in two cases belows:
1. before `up_serialinit` execution
2. in assert

Please use xxxdbg.